src/nni_manager/training_service/pai/paiTrainingService.ts

```diff
@@ -178,7 +177,8 @@ abstract class PAITrainingService implements TrainingService {
         const deferred: Deferred<void> = new Deferred<void>();
 
         request(stopJobRequest, (error: Error, response: request.Response, _body: any) => {
-            if ((error !== undefined && error !== null) || response.statusCode >= 400) {
+            // Status code 202 for success.
+            if ((error !== undefined && error !== null) || response.statusCode !== 202) {
```

Comment:
It's better keep original status code. The HTTP status code is standard. < 400 is normal state. 4xx is permission issue, and  5xx are server errros. Don't hard code to 202 only.

Reply:
There is a case that when submit job to http request, pai returns a 3xx response, but the job is not submitted correctly.

Reply:
3xx means redirection, and it should be handled in most HTTP sdk internally. If it happens here, you can check if the sdk is not used correctly. For example, disabled auto redirect.

Reply:
updated, add `followAllRedirects:true` to handle redirect.